Tipster/Tipster.cpp

```diff
@@ -33,14 +41,144 @@ enum
 
 Tipster::Tipster()
 	:
-	BTextView("TipView")
+	BGroupView("TipView")
 {
-	MakeEditable(false);
-	SetStylable(true);
+	fTipsList = BStringList();
+	fCurrentTip = new BString("");
+	fDelay = 60000000;
+	fReplicated = false;
+	fURL = new BString("");
+	fArtworkTitle = new BString("");
+
+	fTipsterTextView = new TipsterText();
+	fIcon = new BButton("iconview", "", new BMessage(OPEN_URL));
+	fIcon->SetFlat(true);
+	
+	BRect rect(Bounds());
+	rect.top = rect.bottom - 7;
+	rect.left = rect.right - 7;
+	BDragger* dragger = new BDragger(rect, this,
```

Comment:
You should use the constructor without a "rect" argument, as you are in a layouted view.